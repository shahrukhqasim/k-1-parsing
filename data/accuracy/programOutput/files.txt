002 K1 Face Original.json
006 K1 Face Original .json
007 K1 Face Original .json
