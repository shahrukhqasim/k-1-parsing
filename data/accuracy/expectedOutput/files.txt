002 K1 Face Original.words.json
006 K1 Face Original .words.json
007 K1 Face Original .words.json
